standalone/src/models/product-selection/product-selection-draft/fields-config.ts

```diff
@@ -0,0 +1,26 @@
+import { fake, TModelFieldsConfig } from '@/core';
+import { LocalizedString } from '@/models/commons';
+import {
+  TProductSelectionDraftGraphql,
+  TProductSelectionDraftRest,
+} from '../types';
+
+export const restFieldsConfig: TModelFieldsConfig<TProductSelectionDraftRest> =
+  {
+    fields: {
+      name: fake(() => LocalizedString.random()),
+      key: fake((f) => f.lorem.slug()),
+      custom: null,
+      mode: 'Individual',
```

Comment:
since `standalone/src/models/product-selection/constants.ts` exists should we use those constants values here and in line 24?

Reply:
Yes, I think we should, thanks for pointing out.